### DISCUSSION TITLE
What the dead line of project?

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/hack-together-teams#64] [Q&A] [answered]
Hi folks,
Right now our team in active developemnt, we still not submit the project while we not sure how mutch of functlionality we will be able to do, depedning on this it will be different description of project.

What is exact date of dead line for:
- registering? https://forms.microsoft.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR0ktYAUCTtVIvJkJdFsfkalUMjJBTzBYWU1EVVUxWTAyNVcxNTRCS0JXQi4u
- project submittion https://github.com/microsoft/hack-together-teams/issues/new?assignees=&labels=&template=project.yml&title=Project%3A+%3Cshort+description%3E

**Comment by aycabas** [comment]
Hi @YuriyMorozyuk95, deadline of the project submission is June 15th, Midnight PST timezone. You still have time 🙂

**Reply by YuriyMorozyuk95** [reply]
Hi @aycabas.
Then we have time till 15 of June 23:59 PST time?
We are working from Estonia GTM+3, Which means 16 of June 09:59?
Also as I remeber previus time you give to us couple day for recording a video.
It will be possible to provide functional demo video at 19 of June, and technical demo with showing code impementation and azure configuration at 23 of June?

**Reply by aycabas** [reply, marked as answer]
Hi @YuriyMorozyuk95, that's correct it'll be later in Estonia, It'll be 09:59 AM in June 16 for Estonia time.

We will need to get the video recording of your project for the project submission this time before the submission closes.

**Reply by YuriyMorozyuk95** [reply]
Thanks for the answer.
